README.md

```diff
@@ -83,3 +83,17 @@ steps:
         # Required properties
         artifact-download: 'artifacts/foo/bar.txt'
 ```
+
+
+### `kill-before-run` (optional)
+
+Kills all running docker containers if true.
+
+```
+steps:
+  - command: path/to/script.sh
+    plugins:
+      - 'uber-workflow/docker-control':
+        # Required properties
```

Comment:
nit:

> Required properties

says optional above though?

Reply:
Yeah, I think this was just because it's more like a marker of where required properties go. I think this was copied from another Buildkite plugin, but maybe we should refactor these docs.